.storybook/cookie-policy.ts

```diff
@@ -0,0 +1,49 @@
+const COOKIE_ACCEPT_GDPR_NAME = 'GDPR';
+const COOKIE_ACCEPT_VALUE = 'accepted';
+
+function setGDPRCookie(expirationInDays = 365) {
+    const d = new Date();
+    d.setTime(d.getTime() + expirationInDays * 24 * 60 * 60 * 1000);
+    const expires = `expires=${d.toUTCString()}`;
+
+    document.cookie = `${COOKIE_ACCEPT_GDPR_NAME}=${COOKIE_ACCEPT_VALUE};${expires};path=/`;
+}
+
+function hasGdprCookie() {
+    return document.cookie.includes(`${COOKIE_ACCEPT_GDPR_NAME}=${COOKIE_ACCEPT_VALUE}`);
+}
+
+function showGDPRPrompt() {
+    const root = window.document.createElement('div');
+    const rootStyle = window.document.createAttribute('style');
+    rootStyle.value =
+        'position:fixed; background: #333333;width: 100%; bottom: 0; color: antiquewhite;padding: 15px;';
+
+    root.setAttributeNode(rootStyle);
+
+    window.document.body.append(root);
+    root.innerHTML = 'This site uses cookies to understand how you use the site.&nbsp;';
+
+    const acceptButton = window.document.createElement('button');
+    acceptButton.innerHTML = 'Accept Cookies';
+    acceptButton.type = 'button';
+    root.appendChild(acceptButton);
+
+    acceptButton.addEventListener('click', (evt) => {
+        evt.preventDefault();
+        setGDPRCookie();
+        window.location.reload();
+    });
+}
+
+export default function checkAcceptedCookies(): boolean {
+    const gdpr = hasGdprCookie();
+
+    if (gdpr) {
+        setGDPRCookie();
+        return true;
+    }
+
+    showGDPRPrompt();
+    return false;
```

Comment:
not sure about this logic, if you have GDPRcookie, why set it again?

and, also, if you have it but rejected (because you chose that), why show the banner again? if you rejected the cookie, this shouldn't be shown each time you enter the page, right?

Reply:
@pladaria 's idea to make the cookie last forever. there's no reject button: in case you don't accept you'll have the banner untill you accept it (let's keep this simple)

Reply:
well, if we do this for the GRPD law, we need to add the features that the law said, event to reject the cookies once accepted (https://gdpr.eu/cookies/)

Reply:
and we'd need a popup explaining what the cookies  are, what the functionality is for, etc. I don't know, handling the way you say it means another kind of task: no problem, but let's decide this once and for all! 🤔😛